### PR TITLE
Add ORA2 file upload support config

### PR DESCRIPTION
**Pull request by kluo**
@stvstnfrd

### CODE REVIEW

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -1231,6 +1233,10 @@ generic_env_config:  &edxapp_generic_env
       LOCATION: "{{ EDXAPP_CACHE_COURSE_STRUCTURE_MEMCACHE }}"
       # Default to two hours
       TIMEOUT: "7200"
+    ora2_storage:
```
Comment (kluo) [inline playbooks/roles/edxapp/defaults/main.yml]: Technically this is an optional cache that's only used if `ORA2_FILEUPLOAD_BACKEND` is set to 'filesystem'

File: stanford/playground/group_vars/all [hunk 2]
```diff
@@ -16,12 +16,15 @@ EDXAPP_CMS_ENABLE_BASIC_AUTH: True
 EDXAPP_CMS_NGINX_PORT: 80
 EDXAPP_CMS_SITE_NAME: "studio.{{ EDXAPP_SITE_NAME }}"
 EDXAPP_DEFAULT_FROM_EMAIL: "noreply@{{ EDXAPP_LMS_BASE }}"
+EDXAPP_FILE_UPLOAD_STORAGE_BUCKET_NAME: 'edxuploads'
```
Comment (stvstnfrd) [inline stanford/playground/group_vars/all]: Why this value?

File: playbooks/roles/edxapp/defaults/main.yml [hunk 3]
```diff
@@ -624,6 +624,8 @@ EDXAPP_NEWRELIC_WORKERS_APPNAME: "{{ COMMON_ENVIRONMENT }}-{{ COMMON_DEPLOYMENT
 EDXAPP_ORA2_FILE_PREFIX: None
 EDXAPP_FILE_UPLOAD_STORAGE_BUCKET_NAME: '{{ EDXAPP_AWS_STORAGE_BUCKET_NAME }}'
 EDXAPP_FILE_UPLOAD_STORAGE_PREFIX: 'submissions_attachments'
+EDXAPP_ORA2_FILEUPLOAD_BACKEND: None
+EDXAPP_ORA2_FILEUPLOAD_CACHE_NAME: None
```
Comment (stvstnfrd) [inline playbooks/roles/edxapp/defaults/main.yml]: Do we want `None` or the values you added in platform? like `'ora2-storage'`.

Comment (stvstnfrd) [reply playbooks/roles/edxapp/defaults/main.yml]: Also, what happens in devstack vs prod/stage?

Comment (stvstnfrd) [reply playbooks/roles/edxapp/defaults/main.yml]: In fact, do we even need the memcache stuff?

File: playbooks/roles/edxapp/defaults/main.yml (hunk 1 above)
Comment (stvstnfrd) [reply playbooks/roles/edxapp/defaults/main.yml]: Will it break w/out that?

Comment (kluo) [reply playbooks/roles/edxapp/defaults/main.yml]: Yea see https://github.com/edx/edx-ora2/blob/master/openassessment/fileupload/backends/filesystem.py#L66, but I couldn't see a way to append this key to the CACHES dict separately in `stanford/playground/group_vars/all` :/

File: playbooks/roles/edxapp/defaults/main.yml (hunk 3 above)
Comment (kluo) [reply playbooks/roles/edxapp/defaults/main.yml]: Wait yea, actually I think we can just have it use the default django file storage...-.-

File: playbooks/roles/edxapp/defaults/main.yml [hunk 4]
```diff
@@ -624,6 +624,7 @@ EDXAPP_NEWRELIC_WORKERS_APPNAME: "{{ COMMON_ENVIRONMENT }}-{{ COMMON_DEPLOYMENT
 EDXAPP_ORA2_FILE_PREFIX: None
 EDXAPP_FILE_UPLOAD_STORAGE_BUCKET_NAME: '{{ EDXAPP_AWS_STORAGE_BUCKET_NAME }}'
 EDXAPP_FILE_UPLOAD_STORAGE_PREFIX: 'submissions_attachments'
+EDXAPP_ORA2_FILEUPLOAD_BACKEND: 's3'
```
Comment (stvstnfrd) [inline playbooks/roles/edxapp/defaults/main.yml]: Will devstack break if we default to s3?

**Comment by stvstnfrd** [comment]
Cherry-picked to branch...

**Pull request closed without merging**